corehq/apps/custom_data_fields/tests/test_profiles.py

```diff
@@ -1,20 +1,19 @@
 from django.test import TestCase
 
 from corehq.apps.custom_data_fields.models import (
+    PROFILE_SLUG,
     CustomDataFieldsDefinition,
     CustomDataFieldsProfile,
     Field,
-    PROFILE_SLUG,
 )
+from corehq.apps.es.tests.utils import es_test
+from corehq.apps.es.users import user_adapter
 from corehq.apps.users.models import CommCareUser
 from corehq.apps.users.views.mobile.custom_data_fields import UserFieldsView
-from corehq.elastic import get_es_new, send_to_elasticsearch
-from corehq.pillows.mappings.user_mapping import USER_INDEX, USER_INDEX_INFO
 from corehq.pillows.user import transform_user_for_elasticsearch
-from corehq.util.elastic import ensure_index_deleted
-from pillowtop.es_utils import initialize_index_and_mapping
 
 
+@es_test(requires=[user_adapter], setup_class=True)
```

Comment:
> Is there a specific reason for decorating the entire class rather than just the single test method?

No specific reason. This would also result in the creation and deletion of the index only once as it would have in a single test. But I agree that adding it to the test would make it more specific.


>I wonder if the sync_users_to_es decorator would entirely prevent needing to explicitly send the doc to Elasticsearch?

Yep. You are correct. I will update the test accordingly.

Reply:
https://github.com/dimagi/commcare-hq/pull/32522/commits/df1cf80e389f947b73559c54a86a8255884b5218